aws/docker/worker/profile_plot_from_erddap.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import os
+import matplotlib
+matplotlib.use('AGG')
+import matplotlib.dates as mdate
+import boto3
+import cmocean
+import io
+import logging
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
+import netCDF4
+import numpy as np
+import traceback
+from datetime import datetime
+
+from erddapy import ERDDAP
```

Comment:
We'll need to add any new dependencies to https://github.com/ioos/glider-dac-status/blob/master/aws/docker/worker/requirements.txt

Reply:
i will check the dependencies, rename the file, and remove  extraneous comments from the script

Reply:
done